app/import_functions.go

```diff
@@ -960,123 +960,171 @@ func (a *App) importAttachment(data *AttachmentImportData, post *model.Post, tea
 	return fileInfo, nil
 }
 
-func (a *App) importPost(data *PostImportData, dryRun bool) *model.AppError {
-	if err := validatePostImportData(data, a.MaxPostSize()); err != nil {
-		return err
+type PostAndData struct {
```

Comment:
Let's maintain the habit of adding godocs to exported structs :)

Reply:
Yes, in this case the solution is to not export the type because is not needed from the outside.